app/src/lib/feature-flag.ts

```diff
@@ -78,7 +78,7 @@ export function enableTextDiffExpansion(): boolean {
 
 /** Should we allow apps running from Rosetta to auto-update to ARM64 builds? */
 export function enableUpdateFromRosettaToARM64(): boolean {
-  return false
+  return enableBetaFeatures()
```

Comment:
Just to double and triple check this, this function is used in https://github.com/desktop/desktop/blob/57874f8857c7b8d3ef06b1963154ff19abb3dcb5/app/src/ui/lib/update-store.ts#L164-L179

So I ran that code using the base `__UPDATES_URL__` from https://github.com/desktop/desktop/blob/57874f8857c7b8d3ef06b1963154ff19abb3dcb5/script/dist-info.ts#L168-L175

And this was the result:

![image](https://user-images.githubusercontent.com/1083228/116568452-c7b08900-a908-11eb-9d6a-c4b2a23bd533.png)